src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -1971,18 +1995,20 @@ private void doApplicationBegin(
                 doBegin(application, routeId, initialId, traceId, authorization, affinity, beginEx);
                 final MutableInteger value = activeSubscribers.computeIfAbsent(topicFilter, key -> new MutableInteger());
```

Comment:
Suggest using `Int2ObjectHashMap` for active subscribers (and publishers), using `topicKey` approach to get predictable `int` keys for the map.
Note: `HashMap` is functionally correct, but produces more garbage.